### ISSUE TITLE
Named export 'render' not found

### ISSUE THREAD

**Issue by steveninety** [html-to-text/node-html-to-text#305]
### Problem

`import { render } from "dom-serializer"`

The dom-serializer package specifies `export default render`, so importing `{ render }` does not work.

**Occurs where**
Error on build in a SvelteKit / Vite / ESM project.

**Imported in:**
packages/base/src/generic-formatters.js

### Error message
import { render } from "dom-serializer";
         ^^^^^^
SyntaxError: Named export 'render' not found. The requested module 'dom-serializer' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'dom-serializer';
const { render } = pkg;

### Fix
`import render from "dom-serializer"`

Will open a pull request.

**Comment by KillyMXI** [comment]
Hmm.
`dom-serializer` exports `render` both as named function and default:

```js
exports.render = render;
exports.default = render;
```

```js
export function render ...
export default render;
```

Why SvelteKit / Vite can't understand it?

**Comment by KillyMXI** [comment]
> The requested module 'dom-serializer' is a CommonJS module

Why?
Both `html-to-text` version 9 and `dom-serializer` provide both CJS and ESM versions.

**Comment by steveninety** [comment]
Yes it's strange. Will patch it for now, but need to look into it further.

**Comment by KillyMXI** [comment]
My main guess:
Whatever web bundler is used underneath - is not capable enough and falls back to CommonJS for entire dependency subtree once it sees a single CJS package. (There were related issues with Webpack before.)

Proper solutions could be:
- use a smarter bundler or improve the one that is used;
- replace or improve remaining CJS-only packages.

In case of `html-to-text`, there is one remaining - `deepmerge`.
`deepmerge` situation is not great. I should've just written my own package when I had time, as I did with many other things.
There are now
- a "galaxy brain" alternative, or simply too different to figure out whether I can achieve the same behavior I need;
- an updated fork that I'm not sure whether I can trust;
- an idea of more ergonomic package I'm still bearing.

I will choose something when I'm able to resume working on next major version.

**Comment by KillyMXI** [comment]
Finally published the new version - 10.0.0.
Replaced `deepmerge` with `deepmerge-ts`, so it is ESM/CJS dual packages all the way ~~down~~ upstream.
(Not for long - will have to drop CJS in v11, because `htmlparser2` already did this.)

Seems like the issue was also solved already at the Vite side?

I honestly forgot about the PR. Looking at it now, seems like there remains to reason to prefer default import over named.

I consider this solved. Any compatibility issues with version 10 will require a new issue.
